Userland/Libraries/LibGUI/Calendar.cpp

```diff
@@ -39,6 +39,7 @@ Calendar::Calendar(Core::DateTime date_time, Mode mode)
     m_weekend_length = weekend_length;
 
     set_fill_with_background_color(true);
+    set_scrollbars_enabled(false);
```

Comment:
:thinking: there's no content size being set, so I'm not sure what the scrollbar would look like...
what does this look like?